settile.lua

```diff
@@ -0,0 +1,18 @@
+-- sets the selected tile's 'tiletype' ie. changes the tileset you see
```

Comment:
The `tiletypes` command should be able to do this already, right?

Reply:
Probably, I found the tile commands after writing these simple ones. I think I had previously overlooked it(them?) as the script name(s) seemed unrelated to whatever I was doing at the time.